src/Couchbase.Lite.Shared/API/Sync/BasicAuthenticator.cs

```diff
@@ -50,15 +50,15 @@ public sealed class BasicAuthenticator : Authenticator
         /// <summary>
         /// [DEPRECATED] Gets the password that this object holds
         /// </summary>
-        /// [Obsolete("This property is deprecated, please use byte[] PasswordData instead.")]
+        /// [Obsolete("This property is deprecated, please use SecureString PasswordSecureString instead.")]
         [NotNull]
         public string Password { get; }
 
         /// <summary>
         /// Gets the password that this object holds
         /// </summary>
         [NotNull]
-        internal SecureString PasswordSecureString { get; }
+        public SecureString PasswordSecureString { get; }
```

Comment:
Note that the use of SecureString is [no longer recommended by Microsoft](https://github.com/dotnet/platform-compat/blob/master/docs/DE0001.md).  Also this is a public API change, have you consulted @pasin on this?

Reply:
Please use what is recommended by Microsoft.